### ISSUE TITLE
Status of operation

### ISSUE THREAD

**Issue by ThomDietrich** [ThomDietrich/homestead-lara-vedst#1]
homestead-Lara is dependent on a few outside factors, Thereby the setup could fail starting some point in time.

**Current status:**
* 2016-08-01: All fine 
* 2016-12-08: All fine 
* 2017-02-08: All fine

----

Please do not comment here! Open a new issue.

**Comment by ThomDietrich** [comment]
@P1tt187 is the machine still in use?

**Comment by P1tt187** [comment]
I never used it. Don't know if @stefanheyder , @4D44H  or @bedo2991 still do.

**Comment by bedo2991** [comment]
No

**Comment by 4D44H** [comment]
Nope, i use docker containers woth laradock - https://laradock.io/

**Comment by ThomDietrich** [comment]
Thought so. Just wanted to know and have it documented.
The solution in this repo does not work anymore and docker is the better solution now.
